modules/src/ics03_connection/connection.rs

```diff
@@ -1,18 +1,21 @@
 use std::convert::{TryFrom, TryInto};
 use std::str::FromStr;
 
+use serde::Serialize;
```

Comment:
FYI: I often let IntelliJ organize the imports for us. The order makes sense to me:

- standard libs
- non-standard libs
- imports from the same project but different crates
- imports from the same crate

I will continue organizing the imports automatically like this with subsequent PRs in the files I touch.